src/Compilers/Core/Portable/PEWriter/DebugDirectoryExtensions.cs

```diff
@@ -14,28 +14,24 @@ namespace Roslyn.Reflection.PortableExecutable
     // TODO: move to SRM: https://github.com/dotnet/roslyn/issues/24712
     internal static class DebugDirectoryExtensions
     {
-        private static FieldInfo s_dataBuilderField;
-        private static Action<DebugDirectoryBuilder, DebugDirectoryEntryType, uint, uint, int> s_addEntry;
-
-        internal const DebugDirectoryEntryType PdbChecksumEntryType = (DebugDirectoryEntryType)19;
-
-        private static void InitializeReflection()
-        {
-            if (s_dataBuilderField == null)
+        private static Lazy<(FieldInfo, Action<DebugDirectoryBuilder, DebugDirectoryEntryType, uint, uint, int>)> s_debugDirectoryBuilderMembers 
```

Comment:
Consider naming the tuple elements for documentations (dataBuilder and addEntry)